package.json

```diff
@@ -46,6 +46,7 @@
     "build:webpack": "node Makefile.js webpack",
     "build:readme": "node tools/update-readme.js",
     "build:rules-index": "node Makefile.js generateRuleIndexPage",
+    "format": "prettier .",
```

Comment:
we are using trunk everywhere to run these stuffs, Shall we once also check if we can run these through https://docs.trunk.io/code-quality/linters/supported/prettier

Reply:
```plaintext
$ npx trunk check enable prettier
1 linter was enabled:
  prettier (already enabled)
```

Ignoring the glaring bug about calling Prettier a "linter" (🙃) I think it's set up by default. 👍

Reply:
Aha, and TIL from Trunk's messages: `trunk fmt` already exists. Nice.